polxml

```diff
@@ -432,10 +565,10 @@ def main():
                         help = 'input OpenMM xml file (default: field.xml)')
     parser.add_argument('-ox', '--outxml', default = 'field-p.xml',
                         help = 'output OpenMM xml file (default: field-p.xml)')
-    parser.add_argument('-ip', '--inpdb', default = 'config.pdb',
-                        help = 'PDB file with configuration (default: config.pdb)')
-    parser.add_argument('-op', '--outpdb', default = 'config-p.pdb',
-                        help = 'PDB file with configuration (default: config-p.pdb)')
+    parser.add_argument('-ip', '--inpdb', default = 'config.mmcif',
```

Comment:
So you suggest using the pdbx/mmcif as default?  Is this what the majority of users will want? Do the mmcif work will with vmd? 
Woun't it be better to leave the pub as standard and pdbx for the advanced user?

Reply:
My first idea was to consider that a pdbx/mmcif file is written anyway by the fftool, regardless of the size of the system, whereas the pdb is written only for small enough systems.
But I agree with you: the majority of users will use pdb files. And besides mmcif are not readable by VMD...
The option you suggested seems to be the better one!